src/androidTest/java/com/mixpanel/android/mpmetrics/DecideFunctionalTest.java

```diff
@@ -64,7 +64,7 @@ public SharedPreferences get(final long timeout, final TimeUnit unit) throws Int
             }
         };
 
-        mMockConfig = new MPConfig(new Bundle()) {
+        mMockConfig = new MPConfig(new Bundle(), getContext()) {
```

Comment:
Can you put the return value of getContext() in a variable? This call appeared twice in this function (line 27 and 67).